src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -241,6 +242,12 @@ private void processZipFileContent(
 
             InputEnvelope envelope = envelopeProcessor.parseEnvelope(result.getMetadata(), zipFilename);
 
+            log.info(
+                "Parsed envelope. Payment DCNs: {}. Document DCNs: {}",
```

Comment:
worth including zip file name and jurisdiction here ?

Reply:
oops. fixed.